.travis.yml

```diff
@@ -0,0 +1,12 @@
+---
+language: python
+python: "2.7"
+before_install:
+ - sudo apt-get update -qq
+ - sudo apt-get install -qq python-apt python-pycurl
+install:
+  - pip install ansible==1.8.0
+script:
+  - ansible-galaxy install -r tests/requirements.yml
+  - ansible-playbook -i tests/inventory --syntax-check --list-tasks tests/test.yml -e "role_name=ansible-role-sublimetext3_packagecontrol" -e "hosts_group=hosts_group"
```

Comment:
Can you please tell what does it mean:
`-e "role_name=ansible-role-sublimetext3_packagecontrol" -e "hosts_group=hosts_group"`
Have same result output when I cut that.

Looks like rudiment from another role that have `tests/test.yml` with next content:

```yml
---
- hosts: "{{ hosts_group }}"
  roles:
    - role: "{{ role_name }}"
```